src/python/pants_release/start_release.py

```diff
@@ -16,24 +16,48 @@
 
 import requests
 from packaging.version import Version
-from pants_release.git import git, git_fetch
+from pants_release.common import CONTRIBUTORS_PATH, VERSION_PATH, sorted_contributors
+from pants_release.git import git, git_fetch, github_pr_create
+
+from pants.util.strutil import softwrap
 
 logger = logging.getLogger(__name__)
 
+BASE_PR_BODY = softwrap(
+    """
+    This PR is release preparation. Please read over the release notes and make adjustments for
+    clarity for users, such as:
+
+    - shuffle miscategorised items to a better category (including completely uncategorised items)
+
+    - fix or improve poor titles, like "fix bug", so they are more accurate or descriptive (check to
+      see if the PR title is better).
+
+    - fix typos
+
+    Once satisfied, approve and merge, as normal.
+
+    ---
+    """
+)
+
 
 def create_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(description="Prepare the changelog for a release.")
     parser.add_argument(
-        "--prior",
+        "--new",
         required=True,
         type=Version,
-        help="The version of the prior release, e.g. `2.0.0.dev0` or `2.0.0rc1`.",
+        help="The version for the new release, e.g. `2.0.0.dev1` or `2.0.0rc2`.",
     )
     parser.add_argument(
-        "--new",
+        "--release-manager",
         required=True,
-        type=Version,
-        help="The version for the new release, e.g. `2.0.0.dev1` or `2.0.0rc2`.",
+        help="The GitHub username of the person managing this release",
+    )
+    parser.add_argument(
+        "--log-level",
+        default="WARNING",
```

Comment:
For interactive use `WARNING` seems fine, but for running on CI, I was planning to set `--log-level=DEBUG` or something, to make it easier to understand what it's doing.